api/lib/api/performance_manager/vehicle_events.ex

```diff
@@ -0,0 +1,38 @@
+defmodule Api.PerformanceManager.VehicleEvents do
+  use Ecto.Schema
+
+  @primary_key {:pk_id, :id, autogenerate: true}
+  schema "vehicle_events" do
+    # trip identifiers
+    field(:direction_id, :boolean)
```

Comment:
nitpick: this is an integer, even if it's only `0` or `1`.

Reply:
its currently set as a bool in our python schema, so i kept it that way.